cinema/models.py

```diff
@@ -47,8 +55,16 @@
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
-    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(
+        Movie,
+        on_delete=models.CASCADE,
+        related_name="movies"
+    )
+    cinema_hall = models.ForeignKey(
+        CinemaHall,
+        on_delete=models.CASCADE,
+        related_name="movies"
```

Comment:
The `related_name` 'movies' is used for both the `movie` and `cinema_hall` ForeignKey fields in the `MovieSession` model. This can cause conflicts when accessing related objects. Consider using unique `related_name` values for each field to avoid these conflicts.